R/scope.R

```diff
@@ -374,10 +347,8 @@ renv_scope_trace <- function(what, tracer, envir = NULL) {
 
 }
 
-renv_scope_var <- function(key, value, frame = NULL, envir = NULL) {
+renv_scope_var <- function(key, value, frame, envir = parent.frame()) {
```

Comment:
I eliminated the default for `frame` here since I think it's sufficiently important that you should always supply explicitly (and that only changed a single call elsewhere).

Reply:
That sounds reasonable to me.